src/main/java/com/exadel/discount/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package com.exadel.discount.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
```

Comment:
Please add global exception handler for Exception as well if it is not caught on deeper levels

Reply:
+